src/types/algorand-client.ts

```diff
@@ -1,36 +1,21 @@
 import algosdk from 'algosdk'
+import { Config } from '..'
 import { getAlgoNodeConfig, getConfigFromEnvOrDefaults, getDefaultLocalNetConfig } from '../network-client'
 import { TransactionSignerAccount } from './account'
 import { AccountManager } from './account-manager'
 import { AlgoSdkClients, ClientManager } from './client-manager'
-import AlgokitComposer, {
-  AppCallParams,
-  AssetConfigParams,
-  AssetCreateParams,
-  AssetDestroyParams,
-  AssetFreezeParams,
-  AssetOptInParams,
-  AssetTransferParams,
-  MethodCallParams,
-  OnlineKeyRegParams,
-  PayTxnParams,
-} from './composer'
+import AlgokitComposer, { ExecuteParams, MethodCallParams } from './composer'
 import { AlgoConfig } from './network-client'
-import { SendAtomicTransactionComposerResults, SendTransactionFrom } from './transaction'
+import { ConfirmedTransactionResult, SendAtomicTransactionComposerResults, SendTransactionFrom } from './transaction'
+import Transaction = algosdk.Transaction
 
-async function unwrapSingleSendResult(results: Promise<SendAtomicTransactionComposerResults>) {
```

Comment:
I collapsed this method into the class since it was only used in one place and I was able to achieve a more terse syntax.